src/taco-remote/lib/buildManager.ts

```diff
@@ -369,10 +369,20 @@ class BuildManager {
         if (fs.existsSync(changeListFile)) {
             buildInfo.changeList = JSON.parse(fs.readFileSync(changeListFile, { encoding: "utf-8" }));
             if (buildInfo.changeList) {
+                buildInfo.changeList.deletedFiles = buildInfo.changeList.deletedFiles.map(function (deletedFile: string): string {
+                    // Convert all \ and / characters in the path string into platform-appropriate path separators
+                    return deletedFile.split("\\").join(path.sep).split("/").join(path.sep);
```

Comment:
I don't think that the split would break things, but I've changed to a replace since it's a bit nicer anyway.